socorro/middleware/search_common.py

```diff
@@ -80,7 +81,7 @@ def __init__(self, name, default=None, data_type='enum', mandatory=False):
 
 
 class SearchBase(object):
-    meta_filters = [
+    meta_filters = (
```

Comment:
This and related changes are a fix for an issue I discovered while debugging this, where fields were added to ``self.meta_filters`` every time a call to SuperSearch was made, making it increasingly big for nothing.